criu/img-remote.c

```diff
@@ -832,7 +832,7 @@ void accept_image_connections() {
 		int n_events, i;
 
 		n_events = epoll_wait(epoll_fd, events, EPOLL_MAX_EVENTS, 250);
-		if (n_events < 0) {
+		if (n_events < 0 && errno != EINTR) {
```

Comment:
can you add a comment that epoll_wait isn't restarted after interrupted by a signal?

Reply:
http://man7.org/linux/man-pages/man7/signal.7.html